docs/reference/data-streams.asciidoc

```diff
@@ -0,0 +1,115 @@
+[chapter]
+[[data-streams]]
+= Data Streams
+
+You can use data streams to index time-based data that's continuously generated.
+A data stream groups indices from the same time-based data source.
+A data stream tracks its indices, known as _backing indices_, using an ordered
+list.
+
+A data stream's backing indices are <<index-hidden,hidden>>.
+The most recently created backing index is the
+write index. The other backing indices handle read requests. A data stream only
+accepts <<docs-index_,index requests>> with `op_type` set to `create`. To update
+or delete specific documents in a data stream, submit a <<docs-delete,delete>>
+or <<docs-update,update>> API request to the backing index containing the
+document.
+
+To create a data stream, set up a <<indices-templates,composable index
+template>> containing:
+
+* Any desired index patterns in the `index_patterns` property. Indices that
+  match these patterns are in the data stream's _namespace_.
+* A `data_stream` definition that contains the `timestamp_field` property.
+  The `timestamp_field` should be the primary timestamp field 
+   for the data source.
+
+When you index within a data stream's namespace, {es} automatically creates the
```

Comment:
I think "data stream's namespace" here is slightly inaccurate. While a composable template may have a pattern such as `logs-*` that many indexing requests may match, indexing requests themselves must specify a single index or data stream name such as `logs-mysql-prod` or the like. In other words, a data stream has a name but not a namespace, and indexing requests must specify the name.